### PR TITLE
update the time zone convertor example

### PR DESCRIPTION
**Pull request by uyar**

### CODE REVIEW

File: docs/tutorial.rst [hunk 1]
```diff
@@ -331,61 +399,20 @@ that will actually do the conversion:
...
-     <p>Go to the <a href="/">home page</a>.</p>
-   </body>
-   </html>
-
-
 Static assets
 -------------
 
-As our next step, let us apply some style to our markup.
+As our next step, let us apply some style to our page.
```
Comment (mahmoud) [inline docs/tutorial.rst:405]:
```suggestion
As our next step, let us apply some style to our page.
Adding styles will require a CSS file, which should be served without processing.
Such files are generally known as *static assets*.
```
I feel like this explains the term "static" and the fact that it's a general term, not specific to Clastic.

File: docs/tutorial.rst [hunk 2]
```diff
@@ -460,170 +472,120 @@ Working with JSON
...
+    "src_dt": {
+      "text": "Sat Jun 15 21:39:00 2024",
+      "value": "2024-06-15T21:39"
+    },
+    "src_location": "Tijuana"
+  }
+
+Now we arrange the home page template.
+First, we add a modal dialog to display the result.
+By default, it contains dummy information:
```
Comment (mahmoud) [inline docs/tutorial.rst:523]:
```suggestion
By default, it contains placeholder values:
```

**Review by mahmoud — CHANGES_REQUESTED** [review]
Looks great! A marked improvement. A couple of suggestions, open to discussion. Thanks again for this!

File: docs/tutorial.rst [hunk 3]
```diff
@@ -1,40 +1,36 @@
...
-The user will enter a date and a time,
+The user will enter a date and time,
 and select two time zones from a list of all available time zones,
 one for the source location and one for the destination location.
 A screenshot of the final application is shown below.
 
 .. figure:: images/tzconvert_screenshot.*
    :alt: Application screenshot showing the user selected time
-     in Tasmania and Timbuktu.
+     in Tijuana and Timbuktu.
```
[the remark this answers is not in the document]
Comment (uyar) [reply docs/tutorial.rst:22]: Tasmania is not in the Python builtin zoneinfo database.

File: docs/tutorial.rst [hunk 4]
```diff
@@ -53,53 +49,127 @@ so we suggest that you create one for this tutorial.
...
 Getting started
 ---------------
 
-Let's start with an application that just displays the form,
-but doesn't handle the submitted data.
-It consists of a Python source file (``tzconvert.py``)
-and an HTML template file (``home.html``),
+Our first implementation will just display the form;
+it won't handle the submitted data.
+It consists of a Python source file and an HTML template file,
```
[the remark this answers is not in the document]
Comment (uyar) [reply docs/tutorial.rst:69]: That's fine.

[the remark this answers is not in the document]
Comment (uyar) [reply docs/tutorial.rst:69]: Done.

**Comment by mahmoud** [comment]
Looks great, thank you!

**Pull request merged**